src/Repository/CommitteeRepository.php

```diff
@@ -358,6 +358,80 @@ public function countApprovedForReferent(Adherent $referent): int
         ;
     }
 
+    public function findApprovedForReferentAutocomplete(Adherent $referent, $value): array
```

Comment:
Please find a method that tell us what's the goal/intention of this method. It's really unclear what it is doing right now other that it is made for an autocomplete stuff.

Reply:
Same comment for all new methods of this class.

Reply:
It's not easy to find another appropriate name. The goal is to return the values for autocomplete, no other purpose. If I describe what method does it will give `getUuidAndNameOfApprouvedCommitteesSearchedByNameInReferentManagedArea` name, which is not practical. Also there is too little chance that it will be used elsewhere.